src/ledger/LedgerManager.h

```diff
@@ -161,6 +161,11 @@ class LedgerManager
     virtual void deleteOldEntries(Database& db, uint32_t ledgerSeq,
                                   uint32_t count) = 0;
 
+    // getNextBufferedLedger should only be called after checking
+    // hasBufferedLedger
+    virtual bool hasBufferedLedger() const = 0;
+    virtual LedgerCloseData getNextBufferedLedger() = 0;
```

Comment:
this method should probably just be called `popBufferedLedger` - and comment above should just say that it throws if there are no buffered ledgers